### PR TITLE
Counter

### PR DESCRIPTION
**Pull request by efatsi**
Comment counter for articles (and comments as well) works and works quickly. Seeds file updated to generate 0, 2, or 5 comments per generated article. Search feature now displays results for a username search as well.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -22,8 +22,8 @@ def show
   def create
     @comment = @commentable.comments.build(params[:comment].merge(:user_id => current_user.id))
     if @comment.save
-      commenting_on = @comment.commentable_type.pluralize
-      id = @comment.commentable_id
+      parent = @comment.commentable
+      parent.update_count
```
Comment (efatsi) [inline app/controllers/comments_controller.rb:26]: This was messed up when I rebased, this is the comment counter beginning to trickle up though.

File: app/views/articles/index.html.erb [hunk 2]
```diff
@@ -6,8 +6,12 @@
...
 		    <td><h4><%= link_to a.title, a.link %> <%= "(" + a.short_link + ")" %></h4>
 			<%= a.points.to_s + " points by " %> <%= link_to a.username, user_path(a.user) %> <%=", posted " + a.created_at.to_s(:mdy) + " | " %>
-			<%= link_to "#{a.comment_count} comments", article_path(a) %></td>
-		  </tr>
+			<% if a.comment_count == 1 %>
+				<%= link_to "1 comment", article_path(a) %></td>			
+			<% else %>
+				<%= link_to "#{a.comment_count} comments", article_path(a) %></td>
+		  	<% end %>
+		</tr>
```
Comment (efatsi) [inline app/views/articles/index.html.erb:14]: Is there a quicker more ruby-esque way to do this?

File: db/migrate/20120613201704_create_comments.rb [hunk 3]
```diff
@@ -2,6 +2,7 @@ class CreateComments < ActiveRecord::Migration
   def change
     create_table :comments do |t|
       t.text :content
+      t.integer :comment_count, :default => 1
```
Comment (efatsi) [inline db/migrate/20120613201704_create_comments.rb:5]: I've learned the error of my ways. Table adjustments should be done with new migrations.

File: app/views/articles/index.html.erb (hunk 2 above)
Comment (jacobaweiss) [inline app/views/articles/index.html.erb:14]: i think this is what you're looking for? http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize

Comment (efatsi) [inline app/views/articles/index.html.erb:14]: oh awesome, thanks a bunch

**Pull request closed without merging**